include/proposals/proposals_base.hpp

```diff
@@ -5,6 +5,10 @@
 namespace ProposalsCommon {
   constexpr name type_ref_setting = name("r.setting");
```

Comment:
lets call this ref.setting, and ref.code ;)

Reply:
ok, I will change it